velox/type/Type.cpp

```diff
@@ -869,12 +882,19 @@ TypePtr fromKindToScalerType(TypeKind kind) {
 }
 
 void toTypeSql(const TypePtr& type, std::ostream& out) {
-  if (type->isPrimitiveType()) {
-    out << type->toString();
-    return;
-  }
-
   switch (type->kind()) {
+    case TypeKind::SHORT_DECIMAL: {
```

Comment:
for brevity, can combine `case TypeKind::SHORT_DECIMAL, case TypeKind::LONG_DECIMAL` and use
`getDecimalPrecisionScale`